monitoring/tools/utils.py

```diff
@@ -1,8 +1,13 @@
 import gzip
 import logging
+import math
 
 from odc.aws import s3_client, s3_ls_dir, s3_fetch
 
+log = logging.getLogger()
+console = logging.StreamHandler()
+log.addHandler(console)
```

Comment:
Logging in utils is a bad idea.

Whenever it's imported it'll create a logger. Use a function to get a logger, perhaps.

Reply:
actually, it shouldn't have an informative log in an utils.
Removed and added to the function.